server/lobbyconnection.py

```diff
@@ -1073,7 +1073,7 @@ async def command_modvault(self, message):
                                    downloads=downloads, date=int(date.timestamp()), uid=uid, name=name, version=version, author=author,
                                    ui=ui)
                         await self.send(out)
-                    except:
+                    except Exception:
```

Comment:
I mean, I could change it to `BaseException` so that the behavior doesn't change, but I don't think it really makes sense to catch those things. I don't think it matters too much since afaik these commands aren't even used anymore. I'm pretty sure Gatsik told me that he rewrote the py client to use the API.

Reply:
I read up a bit on `BaseException` and I agree

Reply:
I am just now going through this code base for the first time so I have very little context, but I also think this change makes sense - I don't think it's generally good practice to catch `BaseException` unless you have an explicit reason for doing so.